apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -441,7 +439,7 @@ export class RushConfiguration {
    * Example: "C:\MyRepo\common\npm-shrinkwrap.json" or "C:\MyRepo\common\shrinkwrap.yaml"
    */
   public get committedShrinkwrapFilename(): string {
```

Comment:
Given that `5.0` is unreleased, if this change were to be included could the `committedShrinkwrapFilename` and `commonVersions` properties be removed because they will now be confusing?

Reply:
This PR didn't make Rush 5. :-) But we can mark them as `@deprecated` and remove them in Rush 6.

---
In reply to: [197865009](https://github.com/Microsoft/web-build-tools/pull/722#discussion_r197865009) [](ancestors = 197865009)